streams/src/main/java/org/apache/kafka/streams/kstream/internals/ConsumedInternal.java

```diff
@@ -60,7 +60,7 @@ public TimestampExtractor timestampExtractor() {
         return timestampExtractor;
     }
 
-    public Topology.AutoOffsetReset offsetResetPolicy() {
+    public AutoOffsetReset offsetResetPolicy() {
```

Comment:
While this change is fine, it required to also update the code which calls this method.